app/frontend/src/cljs/teet/ui/typography.cljs

```diff
@@ -88,6 +96,8 @@
 
 (def DataLabel (util/make-component :div {:class (<class data-label-style)}))
 
+(def SubSectionLabel (util/make-component :div {:class (<class sub-section-style)}))
```

Comment:
This could be a span or p since div's shouldn't contain text without a wrapping text element. Other than that looks ok